packages/colorpickers/src/styled/ColorPicker/StyledColorWell.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { getColor, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'colorpickers.colorpicker.colorwell';
```

Comment:
It's problematic that the COMPONENT_ID is being used for two components – preventing future theme targeting.